neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
 
-def attention_decoder(decoder_inputs, initial_state, attention_objects,
-                      embedding_size, cell, output_size=None,
-                      loop_function=None, dtype=tf.float32, scope=None):
+from neuralmonkey.logging import debug
 
-    if output_size is None:
-        output_size = cell.output_size
+def attention_decoder(decoder_inputs, initial_state, attention_objects,
```

Comment:
No třeba ten obrázkovej bez attentionu byl, že jo.. Ještě si rozmyslim, jestli je to potřeba jako součást tohodle PR nebo to jde pak dodělat dodatečně.